text_formatter.go

```diff
@@ -154,11 +154,12 @@ func (f *TextFormatter) needsQuoting(text string) bool {
 }
 
 func (f *TextFormatter) appendKeyValue(b *bytes.Buffer, key string, value interface{}) {
-
+	if b.String() != "" {
+		b.WriteByte(' ')
```

Comment:
We apparently allow it to be any buffer (the client can specify it). So it doesn't seem we could do that without breaking the api.